src/util/WellPlateInternal.tsx

```diff
@@ -78,28 +94,46 @@ function GridWellPlateInternal(
       }}
     >
       {[
-        { index: undefined, label: '', isHeader: true },
-        ...columnLabels.map((value) => ({
+        {
           index: undefined,
-          label: value,
+          label: '',
           isHeader: true,
-        })),
+          position: { row: 0, column: 0 },
+        },
+        ...columnLabels.map((value, index) => {
+          return {
+            index: undefined,
+            label: value,
+            isHeader: true,
+            position: {
+              row: 0,
```

Comment:
Didn't we agree to use `-1` for the row in column headers and `-1` for the column in row headers?